plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/annotation/BazelrcFlagAnnotator.kt

```diff
@@ -1,11 +1,11 @@
 package org.jetbrains.bazel.languages.bazelrc.annotation
 
+import com.android.tools.lint.detector.api.startsWith
```

Comment:
i think it's safer to use https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.text/starts-with.html

Reply:
It was not used anyway :). Removed the import. I wonder why the format command didn't remove it.